docs/features/virtual_folders.rst

```diff
@@ -0,0 +1,96 @@
+.. _virtual_folders:
+
+Virtual Folders
+===============
+
+.. versionadded:: 2.7
+
+Virtual folders provide a way to group translations based on any criteria,
+including a file across all the languages in a project, or files on specific
+locations. Virtual folders have priority, so they can be used to allow
+translators to focus on the most important work.
+
+
+.. _virtual_folders#attributes:
+
+Virtual folders attributes
+-----------------------------------
+
+Virtual folders have several attributes:
+
+- A mandatory lowercase name,
+- A mandatory location,
+- An optional priority,
+- An optional browsability flag,
+- An optional description,
+- An optional filtering rule.
+
+Each virtual folder must have a unique combination of name and location. This
+means that there can exist two different virtual folders with the same name if
+they have different locations.
+
+The priority defaults to ``1`` and accepts any value greater than ``0``,
+including numbers with decimals, like ``0.75``.
+
+By default virtual folders cannot be browsed: they won't be displayed, but
+they are still used for sorting.
+
+Also the virtual folders can have a description which might be useful to tell
```

Comment:
*useful to explain the contents of the folder or provide additional instructions*.